modal/_functions.py

```diff
@@ -135,9 +135,11 @@ async def create(
         client: _Client,
         function_call_invocation_type: "api_pb2.FunctionCallInvocationType.ValueType",
     ) -> "_Invocation":
-        assert client.stub
+        io_plane_url = client.server_url  # TODO(nathan): replace with per-function endpoint
+        stub = await ModalClientModal.create(client, io_plane_url)  # TODO(nathan): decide if this is too expensive
```

Comment:
Also wanted to call this out. This is `O(# of gRPC methods)`. I'm assuming this is OK, but it would be good to get confirmation from others.